cloudformation/minimal.yaml

```diff
@@ -602,6 +603,15 @@ Resources:
     Properties:
       CloudFrontOriginAccessIdentityConfig:
         Comment: Comment
+  DBInstance2:
+    Type: AWS::RDS::DBInstance
+    DeletionPolicy: Delete
+    Properties:
+      DBInstanceClass: db.t3.micro
+      AllocatedStorage: 10
+      Engine: mysql
+      MasterUsername: MasterUsername
+      MasterUserPassword: MasterUserPassword
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

Password shouldn't be hardcoded for Resources/DBInstance2/Properties/MasterUserPassword